src/style.css

```
@@ -0,0 +1,125 @@
+body {
+  margin: 0;
+}
+
+.flex-container {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  max-width: 100%;
+}
+
+.block {
+  height: 300px;
+  line-height: 300px;
+  min-width: 300px;
+  text-align: center;
+  font-size: 100px;
+  color: white;
+  font-weight: bold;
+  font-family: Arial;
```

Don't forget to add additional families.
```suggestion
  font-family: Arial, Helvetica, sans-serif;
```